node/src/bin/manager.rs

```diff
@@ -445,6 +442,36 @@ pub enum IndexCommand {
     },
 }
 
+#[derive(Clone, Debug, StructOpt)]
+pub struct FixBlockCommand {
+    /// Network name (must fit one of the chain)
+    #[structopt(empty_values = false)]
+    network_name: String,
+    #[structopt(subcommand)]
+    method: FixBlockSubCommand,
+}
```

Comment:
I find that a little awkward - you'd have to say `graphman fix-block $chain by-hash $hash`; I wonder if it wouldn't be better to make this two subcommands `graphman chain check-blocks [-h hash|-n number|-r range] <chain>` and `graphman chain truncate [-f|--force] <chain>`

Reply:
Fixed in e8794ae5e

Reply:
Fixed in e8794ae5e